### PR TITLE
Bring in CA app template

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

Sets up the project with the CA app template in preparation for development. We are on a tight project timeline and setting this work up now saves development time down the road while we are still in the design phase.

I committed to `develop` the PR template, and issue templates.

Connects #1

### Demo

Screen Shot 2019-03-26 at 4 52 43 PM

## Testing Instructions
A default CRA app should display at http://localhost:3474 after VM setup and starting the server

`./scripts/setup`
`vagrant ssh`
`./scripts/server`

### CODE REVIEW

**Comment by caseycesari** [comment]
When running `./scripts/setup`, I get the following error: 

```
==> default: Running provisioner: shell...
    default: Running: inline script
==> default: Running provisioner: ansible...
`playbook` does not exist on the host: /Users/ccesari/projects/pwd-fishway/deployment/ansible/pwd-fishway.yml
```

Maybe just missing a rename?

**Comment by fungjj92** [comment]
Missed that one, thanks! Pushed up the file rename.

**Comment by caseycesari** [comment]
Thanks! Getting a new error, along with a handy message from Ansible.

```
ERROR! the role 'pwd-fishway.docker' was not found in /Users/ccesari/projects/pwd-fishway/deployment/ansible/roles:/Users/ccesari/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:/Users/ccesari/projects/pwd-fishway/deployment/ansible

The error appears to have been in '/Users/ccesari/projects/pwd-fishway/deployment/ansible/pwd-fishway.yml': line 12, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    - { role: "azavea.python-security" }
    - { role: "pwd-fishway.docker" }
      ^ here
This one looks easy to fix.  It seems that there is a value started
with a quote, and the YAML parser is expecting to see the line ended
with the same kind of quote.  For instance:

    when: "ok" in result.stdout

Could be written as:

   when: '"ok" in result.stdout'

Or equivalently:

   when: "'ok' in result.stdout"

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by fungjj92** [comment]
Let me go through the full project set up again. I had set up the project before renaming the `project-name` throughout, which has resulted in issues!

**Comment by fungjj92** [comment]
Pushed up fixes. `ctrl+f` doesn't search filenames as it turns out 🤔Sorry bout that. So quick to destroy and rebuild the VM though.

**Comment by caseycesari** [comment]
Did a fresh pull and destroyed my VM, but still getting the second error I pasted. I tried messing with the file it pointed out, but I couldn't fix it. The syntax looks fine to me.

**Comment by caseycesari** [comment]
It maybe looks like this folder didn't get added in this PR: https://github.com/azavea/civic-apps-app-template/tree/develop/deployment/ansible/roles.

You may have it locally though.

**Comment by fungjj92** [comment]
Bested by `git add`. Pushed up.

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  config.vm.provider :virtualbox do |vb|
+    vb.memory = 2048
+    vb.cpus = 2
+  end
+
+  # Civic Apps Port Mappings
+  # Uncomment those you need, delete those you don't
+  # Remove this comment when done
+
+  # Nginx
```
Comment (caseycesari) [inline Vagrantfile]: I think we are safe to remove these.

File: scripts/cibuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [[ -n "${project_name_DEBUG}" ]]; then
```
Comment (caseycesari) [inline scripts/cibuild]: Looks like `project_name` needs to be updated in all the scripts.

**Review by caseycesari — APPROVED** [review]
Looks good!

**Comment by fungjj92** [comment]
Thanks for the patient review :) !

**Pull request merged**